src/main/kotlin/com/sourcegraph/cody/chat/ChatSession.kt

```diff
@@ -39,7 +42,7 @@ interface ChatSession {
 }
 
 class AgentChatSession
-private constructor(private val project: Project, newSessionId: CompletableFuture<SessionId>) :
+private constructor(private val project: Project, private val internalId: String, newSessionId: CompletableFuture<SessionId>, ) :
```

Comment:
Can we move `internalId` to be last param and add it default with UUID?
I know the si early preview, just adding comments to not forgot later.

Reply:
Fixed.